cdk_emqx_cluster/cdk_emqx_cluster_stack.py

```diff
@@ -46,6 +47,74 @@
 with open("./ssm_docs/start_traffic.yaml") as f:
     doc_start_traffic = f.read()
 
+def loadgen_setup_script(n : int, hostname: str) -> str:
+    return textwrap.dedent(
+        f"""\
+        cat <<EOF > /usr/bin/loadgen-setup.sh
+        #!/bin/bash
+        set -xeu
+
+        hostname {hostname}
+        hostnamectl set-hostname {hostname}
+
+        for x in \$(seq 2 250); do ip addr add 192.168.{n}.\$x dev ens5; done
+
+        # emqtt bench escript will not start epmd, so we start it here
+        epmd -daemon
+
+        touch /tmp/setup-done
+        EOF
+
+        chmod +x /usr/bin/loadgen-setup.sh
+
+        cat <<EOF > /etc/systemd/system/loadgen.service
+        [Unit]
+        Description=Configures loadgen on every boot
+
+        [Service]
+        ExecStart=/bin/bash /usr/bin/loadgen-setup.sh
+
+        [Install]
+        WantedBy=multi-user.target
+        EOF
+
+        systemctl daemon-reload
+        systemctl enable loadgen.service
+        systemctl start loadgen.service
+        """
+    )
+
+def emqx_setup_script(n : int, hostname: str) -> str:
+    return textwrap.dedent(
+        f"""\
+        cat <<EOF > /usr/bin/emqx-setup.sh
+        #!/bin/bash
+        set -xeu
+
+        hostname {hostname}
+        hostnamectl set-hostname {hostname}
+
+        touch /tmp/setup-done
+        EOF
+
+        chmod +x /usr/bin/emqx-setup.sh
+
+        cat <<EOF > /etc/systemd/system/emqx-setup.service
+        [Unit]
+        Description=Configures EMQX on every boot
+
+        [Service]
+        ExecStart=/bin/bash /usr/bin/emqx-setup.sh
+
+        [Install]
+        WantedBy=multi-user.target
```

Comment:
I guess it ensures this runs before emqx starts, right?

Reply:
I guess that this by itself won't guarantee that this service is run before emqx, because it seems that emqx also has the same WantedBy:

```sh
ubuntu@emqx-1:~$ systemctl show -p WantedBy emqx
WantedBy=graphical.target multi-user.target

ubuntu@emqx-1:~$ systemctl list-dependencies --before emqx-setup.service
emqx-setup.service
● ├─multi-user.target
● │ ├─cloud-final.service
● │ ├─systemd-update-utmp-runlevel.service
● │ ├─cloud-init.target
● │ │ └─shutdown.target
● │ ├─graphical.target
● │ │ ├─systemd-update-utmp-runlevel.service
● │ │ └─shutdown.target
● │ └─shutdown.target
● └─shutdown.target
```

I'll add a `Before=` to this one to make it start before emqx.

(Trying out manually seems to work:
```sh
ubuntu@emqx-1:~$ systemctl list-dependencies --before emqx-setup.service
emqx-setup.service
● ├─emqx.service
● ├─multi-user.target
● │ ├─cloud-final.service
● │ ├─systemd-update-utmp-runlevel.service
● │ ├─cloud-init.target
● │ │ └─shutdown.target
● │ ├─graphical.target
● │ │ ├─systemd-update-utmp-runlevel.service
● │ │ └─shutdown.target
● │ └─shutdown.target
● └─shutdown.target
```
)